[DO NOT MERGE] DOCS-12770 lifecycle states (#166)
mulesoft/docs-exchange: modules/ROOT/pages/asset-types.adoc

= Asset Types
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> 

Reviewer: GLOBALLY replace space-hyphen-space with a colon.

The following asset types are supported in Exchange:

> * Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols

Reviewer: ```suggestion
* Connectors: Packaged connectivity to an Anypoint Platform endpoint using third-party APIs and standard integration protocols
```

+
Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.

> * Templates - Packaged integration patterns built on best practices to address common use cases

Reviewer: ```suggestion
* Templates: Packaged integration patterns that implement best practices to address common use cases
```

+

> You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.

Reviewer: ```suggestion
Complete the template's use case or solution by supplying your own information, customizing or extending the templates as needed.
```

* Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security
* REST APIs - A RAML file or OAS file that specifies an API
+

> These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.

Reviewer: ```suggestion
* REST APIs: RAML or OAS files that specify APIs referenced by an HTTP Request connector to expose metadata to Anypoint Studio
```

* AsyncAPI specifications - An AsyncAPI specification file that specifies an event-driven API
* SOAP APIs - A WSDL file that specifies an API
* HTTP APIs - A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager
* API Groups - A set of APIs bundled into a single asset
+
Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.

> * API Spec Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification

Reviewer: ```suggestion
* API Spec Fragments - A RAML document that has a version and an identifier but is not in itself a complete RAML specification
```

+

> API spec fragments are also known as RAML fragments.

Reviewer: ```suggestion
API spec fragments are also known as _RAML fragments_.
```
